disnake/ext/commands/slash_core.py

```diff
@@ -468,6 +508,18 @@ def description(self) -> str:
     def options(self) -> List[Option]:
         return self.body.options
 
+    @property
+    def name_localizations(self) -> LocalizationValue:
+        return self._name_localised.localizations
+
+    @property
+    def description_localizations(self) -> LocalizationValue:
+        return self._description_localised.localizations
+
+    add_option = SlashCommand.add_option
```

Comment:
I'm confused on the point of this, this class inherits from `SlashCommand` right?

Reply:
https://github.com/DisnakeDev/disnake/pull/666/commits/c83a0212aabb32bdcf7d4085a386efb2972a2986